webpack/ForemanTasks/Components/TasksTable/TasksTableHelpers.js

```diff
@@ -10,3 +10,21 @@ export const getApiPathname = () => {
   const uri = new URI(window.location.href);
   return uri.pathname().replace('foreman_tasks/', 'foreman_tasks/api/');
 };
+
+export const resolveSearchQuery = search => {
+  const uriQuery = {
+    search,
+    page: 1,
+    per_page: 20,
+  };
+  updateURlQuery(uriQuery);
+};
+
+export const getSubTaskID = () =>
+  window.location.pathname.split('/').slice(-2)[0];
```

Comment:
I think you should be able to receive this value from the react-router `location` prop.
See: https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/withRouter.md

Reply:
Changed, thanks